packages/rancher-istio/charts/app-readme.md

```diff
@@ -10,6 +10,11 @@ For more information on how to use the feature, refer to our [docs](https://ranc
 #### Airgapped Environments
 If you are using this chart in an airgapped environment, you will not be able to upgrade. This is because the `istioctl` upgrade command reaches out to an external repo and it is not configurable. We are tracking the fix for this issue [here](https://github.com/rancher/rancher/issues/33402)
 
+#### Installing Istio with CNI component enabled on RHEL 8.4 SElinux enabled cluster.
+To install istio with CNI enabled, e.g. when cluster has a default PSP set to "restricted", on a cluster using nodes with RHEL 8.4 SElinux enabled, run the following command on each cluster node before creating a cluster.
+`mkdir -p /var/run/istio-cni && semanage fcontext -a -t container_file_t /var/run/istio-cni && restorecon -v /var/run/istio-cni`
+See [this issue](https://github.com/rancher/rancher/issues/33291) for details.
```

Comment:
Hmm this doesn't seem to be the right way to approach this. Files in `/var/run/istio-cni` should not be labeled as type `container_file_t` since that implies that **any** container should be able to access these files. Is that correct, or are there specific processes that need to have permissions to access files in this directory?

The normal approach we would take with adding support for SELinux for a chart would be to introduce a new type into our RPM (e.g. https://github.com/rancher/rancher-selinux/pull/11) that allows it to read this type of file (which can be labeled [here](https://github.com/rancher/rancher-selinux/blob/1384bec84b55e912c7ae8db48c87767faca07b89/policy/centos8/rancher.fc#L1-L2). It would be expected that nodes in the cluster that have SELinux have this type installed if they followed our docs (re: https://rancher.com/docs/rancher/v2.6/en/security/selinux/).

After that, any container within the pod that needs permissions to read those files will be able to access them by providing them with the appropriate options, which also needs to be included in the PSP (see changes introduced in the subchart for Monitoring [here](https://github.com/rancher/charts/pull/1766/commits/145196cf6684ff43fd8ade0890d3c9f8920710b7) and the corresponding changes to set values on that chart [here](https://github.com/rancher/charts/pull/1766/commits/e5ce500071c9830d5a3e9da26ddf874cf2e8ac4a#diff-430c2d604a1da58b9127a7a43e997b51435f7de1c614b1a6a618e431467876d5R95)

Reply:
For now, I think we should file this as a new issue to figure out how to add SELinux support onto Istio, release note it, and remove this from the README.md entirely for this PR since that workaround should not be our recommended approach.

Reply:
@aiyengar2 Thanks for the insights. We'll look into it. For your context, this known issue & workaround were added for [issue# 33291](https://github.com/rancher/rancher/issues/33291). [This comment](https://github.com/rancher/rancher/issues/33291#issuecomment-1021298202) explains the details behind the change. 
If we were to not add it to README file here, that would also mean we won't have any fix/workaround for that issue in v2.6.4